resource-aggregate/pb/events.proto

```diff
@@ -113,4 +113,32 @@ message ResourceStateSnapshotTaken {
     repeated ResourceDeletePending resource_delete_pendings = 6;
     AuditContext audit_context = 100;
     EventMetadata event_metadata = 101;
+}
+
+message DeviceMetadataUpdated {
+    string device_id = 1;
+    oneof updated {
+        ConnectionStatus status = 2;
+        ShadowSynchronizationStatus shadow_synchronization_status = 3;
+    };
+    AuditContext audit_context = 100;
+    EventMetadata event_metadata = 101;
+}
+
+message DeviceMetadataUpdatePending {
+    string device_id = 1;
+    oneof update_pending {
+        ShadowSynchronizationStatus shadow_synchronization_status = 2;
```

Comment:
Skip status, just ShadowSynchronization.

Reply:
`shadow_synchronization_status` you want to skip it for all messages ?

Reply:
fixed